pyndl/preprocess.py

```diff
@@ -351,7 +379,19 @@ def process_context(line):
                 process_words(words)
 
 
-class JobFilter():
+class JobFilterBase():
```

Comment:
What is the reasoning behind defining this base class? Is it to define the interface?

Reply:
Yes. The type checker requires method definitions, but doesn't accept assignment already existing methods.